packages/core/src/types.ts

```diff
@@ -835,7 +841,7 @@ export interface StateNodeConfig<
     TDelay
   >;
   /**
-   * @private
+   * @internal
```

Comment:
We are using `stripInternal` and I'm not sure if we want to "hide" this parent property like this. I'd remove this particular comment completely.